src/components/Accordion/AccordionItem/AccordionItem.scss

```diff
@@ -95,3 +98,18 @@ $accordionHeader_height: 48px;
         background-color: var(--color-dark60);
     }
 }
+
+// Animation
+.accordionItem_opened {
```

Comment:
please follow conventions for modifiers - .accordionItem.opened  (  .accordion { &.opened {} }